src-docs/src/views/markdown_editor/markdown_editor_with_plugins.js

```diff
@@ -261,7 +268,7 @@ Notice the toolbar above has a new chart button. Click it to add a chart.
 
 Once you finish it'll add some syntax that looks like the below.
 
-!{chart{"palette":4,"height":300}}
+!{chart{"palette":2,"categories":5}}
```

Comment:
```suggestion
!{chart{"palette":"2","categories":5}}
```

The `2` needs to be wrapped in quotes now.